src/main/java/seedu/address/model/person/Gender.java

```diff
@@ -0,0 +1,81 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.StringJoiner;
+
+/**
+ * Represents a Person's gender in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidGender(String)}
+ */
+public class Gender {
+
+    public static final String DATA_TYPE = "Gender";
+    public static final String MALE = "Male";
+    public static final String FEMALE = "Female";
```

Comment:
Can make this into regex form and accept both capitalised and non-capitalised inputs